src/runtime/vendor/github.com/fsnotify/fsnotify/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
It might have been better to have a separate commit for the addition of `fsnotify`. Not worth a respin.

Reply:
I wanted to ensure to have the proper vendored code in the same commit, but yeah, it makes harder to get the actual code changed. Will split next time.